### DISCUSSION TITLE
GitHub Copilot ERROR

### DISCUSSION THREAD

**Discussion by sh0z3n** [community/community#42507] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

GitHub Copilot IS SENDING ME ERROR MESSAGE ABOUT SERVER CONNECTION !!! ALTHOUGH MY SHCOOL PAID FOR ME A SUBSCRIBTION HELP !

**Comment by Akash1134** [comment, marked as answer]
Hi @dev-Mokhtar , sorry for the trouble, have you tried to verify your account [here](https://education.github.com/discount_requests/pack_application) in order to make sure you've got the access? If that doesn't help, please refer to the[ troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot) and let us know how it goes. Thanks!

**Reply by sh0z3n** [reply]
I did it nothing Changed the same error message of server This is really shameful for Such a company like github ....
